File: 2-semestre/uri-online-judge/c/1017.c
#include <stdio.h>
 
int main() {
    double tempoGasto, veloMedia, litrosNec;
    
    scanf("%lf", &tempoGasto);
    scanf("%lf", &veloMedia);
    litrosNec = (tempoGasto * veloMedia) / 12;
    
    printf("%.3lf\n", litrosNec);
    
    return 0;
}

File: 2-semestre/uri-online-judge/javascript/1017.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let tempoGasto = parseInt(lines.shift());
let veloMedia = parseInt(lines.shift());
let litrosNec = (tempoGasto * veloMedia) / 12;

console.log(litrosNec.toFixed(3));
